CHANGELOG.md

```diff
@@ -6,6 +6,68 @@ For advice on how to use these release notes see [our guidance on staying up to
 
 ### Breaking changes
 
+#### Use the `govuk-applied-colour` function to access applied colours in Sass
+
+We've restructured our applied colours in Sass, changing how you can access applied colours for use in your own code.
+
+If you were using the value of one of the variables in [`settings/_colours-applied.scss`](https://github.com/alphagov/govuk-frontend/blob/v5.13.0/packages/govuk-frontend/src/govuk/settings/_colours-applied.scss), use the new `govuk-applied-colour` function to access that value.
+
+```scss
+.a-component {
```

Comment:
Let's use something other than `a-component` - as these don't have to be contained within a component, and the `a` is making me think of `<a>` tags.

Could just use `.selector` or something.